final_prep/mod_zero_hero.rb

```diff
@@ -56,16 +125,60 @@
 #   - luckyNumbers (array)
 #   - address (hash with following key/values: number , street , state, zip)
 
+scary_monster = {
+  name: "Bob",
+  smell: "rotten apples",
+  weight: 300,
+  citiesDestroyed: ["Cleveland", "Sacramento", "Topeka"],
+  luckyNumbers: [13, 66, 0],
+  address: {
+    number: 23,
+    street: "Penny Lane",
+    state: "Kansas",
+    zip: 66221
+  }
+}
+
 
 # Create a new class called SuperHero
 # - Your class should have the following DYNAMIC values
-#   - name 
+#   - name
 #   - super_power
-#   - age 
+#   - age
 # - Your class should have the following STATIC values
 #   - arch_nemesis, assigned to "The Syntax Error"
 #   - power_level = 100
-#   - energy_level = 50 
+#   - energy_level = 50
+
+class SuperHero
+  attr_reader :name, :super_power, :age, :number
+  def initialize(name, super_power, age, number)
+    @name = name
+    @super_power = super_power
+    @age = age
+    @number = number
+  end
+
+  def arch_nemesis
+    "The Syntax Error"
+  end
+
+  def power_level
```

Comment:
`power_level` and `energy_level` are supposed to be variables on the class, not methods (functions).  You would define them much the same way you define a variable in Ruby in general. ie -

```
@arch_nemesis = "The Syntax Error"
@power_level = 100
@energy_level = 50
```